src/main/java/com/exasol/adapter/request/parser/PushdownSqlParser.java

```diff
@@ -637,8 +689,8 @@ private ColumnMetadata findColumnMetadata(final String tableName, final String c
                 return columnMetadata;
             }
         }
-        throw new IllegalStateException("Could not find column metadata for involved table " + tableName
-                + " and column + " + columnName + ". All involved tables: " + this.involvedTablesMetadata.toString());
+        throw new IllegalStateException("Could not find column metadata for involved table \"" + tableName
+                + "\" and column \"" + columnName + "\". All involved tables: " + this.involvedTablesMetadata.toString());
```

Comment:
There is another `IllegalStateException` in `findInvolvedTableMetadata` without quotes around names.